src/Microsoft.SqlTools.ServiceLayer/EditData/UpdateManagement/RowDelete.cs

```diff
@@ -74,6 +100,38 @@ public override DbCommand GetCommand(DbConnection connection)
             return command;
         }
 
+        /// <summary>
+        /// Runs a query using the where clause to determine if duplicates are found (causes issues when deleting).
+        /// </summary>
+        private bool checkWhereDuplicate(WhereClause where, string input, DbConnection connection)
+        {
+            bool verifyStatus = false;
+            using (DbCommand command = connection.CreateCommand())
+            {
+                command.CommandText = input;
+                command.Parameters.AddRange(where.Parameters.ToArray());
+                using (DbDataReader reader = command.ExecuteReader())
+                {
+                    try
+                    {
+                        while (reader.Read())
+                        {
+                            if (reader.GetInt32(0) != 1)
+                            {
+                                verifyStatus = true;
```

Comment:
I'd just return true here and get rid of this var completely (returning false at the end of the method)